eclipsePlugin/build.gradle

```diff
@@ -102,18 +183,15 @@ def distSpec = copySpec {
     include 'doc/*.*'
     include 'plugin/*.*'
     include 'icons/*.*'
+    include 'lib/'
   }
-  from('../findbugs/') { // FIXME : Ugly... but those plugin are gonna die anyway...
+  from('../spotbugs/') { // FIXME : Ugly... but those plugin are gonna die anyway...
     include 'plugin/**'
   }
-  from('../findbugs/optionalPlugin/noUpdateChecks.jar') { // FIXME : ugly, but update checks are gonna die too
+  from('../spotbugs/optionalPlugin/noUpdateChecks.jar') { // FIXME : ugly, but update checks are gonna die too
     into 'plugin/'
   }
-  from(configurations.includedLibs) {
-    into 'lib/'
-    rename 'annotations-.*', 'annotations.jar'
-  }
-  from('META-INF/MANIFEST.MF') {
+  from('META-INF/MANIFEST-DIST.MF') {
     filter { line ->
```

Comment:
we probably don't need this anymore, since the manifest is being updated before creating the jar, and the version is set there